Makefile

```diff
@@ -2,28 +2,40 @@
 
 include Makefile.include
 
+ifeq ($(PROFILES),)
+PROFILES := 'pmm'
+endif
+
 env-up: 							## Start devcontainer.
+	COMPOSE_PROFILES=$(PROFILES) \
```

Comment:
do we really need to set it for each item?
can we extract it outside of items?

Reply:
I need to pass default profile (`pmm`), for some reason it does'n work with default
<img width="941" alt="image" src="https://user-images.githubusercontent.com/91831719/196446628-62430203-708e-4f8a-a5a2-4d7c12b40204.png">

<img width="1184" alt="image" src="https://user-images.githubusercontent.com/91831719/196446870-c05152a3-0513-4056-bc7b-711fce3ba6ca.png">